internal/rukpak/convert/registryv1.go

```diff
@@ -195,17 +195,17 @@ func Convert(in RegistryV1, installNamespace string, targetNamespaces []string)
 	for _, depSpec := range in.CSV.Spec.InstallStrategy.StrategySpec.DeploymentSpecs {
 		annotations := util.MergeMaps(in.CSV.Annotations, depSpec.Spec.Template.Annotations)
 		annotations["olm.targetNamespaces"] = strings.Join(targetNamespaces, ",")
+		depSpec.Spec.Template.Annotations = annotations
 		deployments = append(deployments, appsv1.Deployment{
 			TypeMeta: metav1.TypeMeta{
 				Kind:       "Deployment",
 				APIVersion: appsv1.SchemeGroupVersion.String(),
 			},
 
 			ObjectMeta: metav1.ObjectMeta{
-				Namespace:   installNamespace,
-				Name:        depSpec.Name,
-				Labels:      depSpec.Label,
-				Annotations: annotations,
```

Comment:
@joelanford, I have a quick question: should we keep the annotations in both places, given that we’re merging other annotations with annotations := util.MergeMaps(in.CSV.Annotations, depSpec.Spec.Template.Annotations)?

Could we not have more annotations ?

Reply:
I'm just trying to match OLMv0's behavior (which I may _still_ not have exactly right, btw. It's complicated). We are also dealing with the etcd size limit, so I don't want to duplicate stuff more than necessary.  There is a `Subscription.spec.config.annotations` in OLMv0 that I think propagates to the deployment's annotations. So once we have the parameterized value stuff design/implemented/supported, users will be able to provide custom annotations.

Reply:
> dealing with the etcd size limit
Bit by bit, memories are resurfacing... I remember this

Make sense 👍